gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-bin.zip
+zipStoreBase=GRADLE_USER_HOME
+distributionSha256Sum=6ac2f8f9302f50241bf14cc5f4a3d88504ad20e61bb98c5fd048f7723b61397e
```

Comment:
sorry for the might be stupid question, what is this sha for?

Reply:
I copied it from upstream, I guess it's an extra check to make sure that binary downloaded is valid (probably security concerns?)